src/github_services.py

```diff
@@ -76,17 +82,18 @@ def _get_request_headers() -> Dict[str, str]:
 def get_prs_assigned_to_reviewers(
     org_name: str,
     repository: str,
-    wait_hours: int
+    max_waiting_time: int
```

Comment:
The name of this variable needs to have a unit in it.

Reply:
updated